pantheon-bundle/src/main/java/com/redhat/pantheon/model/assembly/Assembly.java

```diff
@@ -0,0 +1,21 @@
+package com.redhat.pantheon.model.assembly;
+
+import com.redhat.pantheon.model.api.Child;
+import com.redhat.pantheon.model.api.Field;
+import com.redhat.pantheon.model.api.WorkspaceChild;
+import com.redhat.pantheon.model.api.annotation.JcrPrimaryType;
+import com.redhat.pantheon.model.module.Module;
+
+import javax.inject.Named;
+import java.util.Locale;
+
+@JcrPrimaryType("pant:assembly")
+public interface Assembly extends Module {
```

Comment:
Why does Assembly extend Module? Are we sure that it should? Perhaps it would make more sense for both Assembly and Module to extend a common parent.

Reply:
And for that matter, maybe Resource should extend from the parent as well, depending on what turns out to be shared.

Reply:
+1

Reply:
who is this common parent :-)?

Reply:
Well, we'd have to create one. :)

Maybe create a `Document` class?

Reply:
Or a `Content` class instead? Certainly open to suggestions here.

Reply:
Content class is taken. https://github.com/redhataccess/pantheon/blob/master/pantheon-bundle/src/main/java/com/redhat/pantheon/model/module/Content.java

Reply:
Then Document seems like a fitting choice, unless anyone else has a 3rd option!

Reply:
I implemented a generic Document class as a parent for Module and Assembly. I also add a DocumentLocale, which is a Child of Document. Let me know what guys think...